Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Proteinortho is a tool to detect orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0b/proteinortho-v6.0b.tar.gz"
```

Comment:
This is marked as a beta release, we don't ship unstable versions: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
I declared it as beta, since i did not published it. But is tested intensively and stable. I will remove the b.